compose/desktop/desktop/build.gradle

```diff
@@ -78,6 +78,10 @@ tasks.findByName("jvmTest").configure {
     systemProperties["GOLDEN_PATH"] = getGoldenPath(project).toString()
 }
 
+tasks.withType(Test).configureEach {
+    maxParallelForks = 1
```

Comment:
Isn't `1` default value?

PS I experimented with it some time ago in opposite way - #529 😅

Reply:
We can't run tests in parallel because it can fail tests that take screenshots or rely on focus.

Reply:
Yep, exactly